go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/agent-sdk-go
+
+go 1.14
+
+require (
+	github.com/OSSystems/pkg v0.0.0-20170629204819-0c38fb3a1f0f // indirect
+	github.com/UpdateHub/agent-sdk-go v0.0.0-20190821190221-9c34ece2ec09 // indirect
+	github.com/UpdateHub/updatehub v0.0.0-20200306165828-746469e2adfb
```

Comment:
This seems wrong. It did not find the tag.

Reply:
Go mod uses the [semver](https://semver.org/) to find the version of the package, usually in the form **v**(major).(minor).(patch), so the **v** is required to fetch the right version, but the current version of the updatehub is `1.1.1` not `v1.1.1`. That is the reason to cause this mismatch on tag version.

Reply:
Although that the package is right

Reply:
Ok; it is ridiculous it requires the tag to have `v` prefix but it indeed seems to be the case.